src/constants/networks.ts

```diff
@@ -258,7 +258,7 @@ const AuroraNetworkInfo: NetworkInfo = {
   name: 'Aurora',
   imageURL: AURORA_LOGO_URL,
   client: createClient('https://api.thegraph.com/subgraphs/name/viet-nv/kyberswap-elastic'), //todo namgold: change this
```

Comment:
change this to https://api.thegraph.com/subgraphs/name/kybernetwork/kyberswap-elastic-aurora 

we havent deploy yet, but subgraph format can be pre-defined.

Reply:
sure